### DISCUSSION TITLE
[Feature] Add aspect ratio in the search options

### DISCUSSION THREAD

**Discussion by lucianosrp** [immich-app/immich#9098] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometime I find myself searching for photos with a specific aspect ratio. 
For example, I would like to search for all photos in 16:9 for my new Desktop Wallpaper or in Portrait for my new phone lockscreen.
It would be nice to have an option to filter assets based on aspect ratio in the search bar's search option. 
Options like:

- Portrait (9:16)
- Square (1:1)
- Landscape (16:9)

I guess could be a good feature to have! 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by jteez** [comment]
Great Idea, in its simplest form allow to search in just Landscape or Portrait. Usecase, I'm looking to print some portrait photos of birds, I can search context 'birds' but would be good to add 'landscape'

**Comment by Ilpredatore** [comment]
I thought a the same use case. is this function in roadmap for future release? thank you very much

**Comment by patric-r** [comment]
+1

**Comment by networxnet** [comment]
Any update here? I'm a little surprised that such a basic feature isn't available even in Version 3. 🤔

**Comment by stantheblan** [comment]
bump

**Reply by bo0tzz** [reply]
Please don't leave comments like this, they're basically just spam and unnecessarily notify everyone following the thread. If you want to support a feature request, use the upvote button on the main post.
